design/20240122.scarf.md

```diff
@@ -0,0 +1,62 @@
+# Proposal: Scarf.sh integration
+
+<!-- toc -->
+- [Summary](#summary)
+  - [What is Scarf?](#what-is-scarf)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+- [Proposal](#proposal)
+  - [How will this change impact our users?](#how-will-this-change-impact-our-users)
+  - [Known issues/limitations](#known-issueslimitations)
+<!-- /toc -->
+
+## Summary
+
+With our focus on CNCF graduation, CNCF aims for its projects to become [vendor-neutral](https://contribute.cncf.io/maintainers/community/vendor-neutrality/) wherever possible. The cert-manager project should uphold this aim. In doing so, it will need to take a further step to move on from its proud Jetstack legacy with a change to remove Jetstack from the container image repository name.
```

Comment:
Scarf.sh does not solve this problem, as explained in the "known issues/limitations" section below.
> Currently, the Scarf service only allows for custom domains and doesn't include custom paths.

The word Jetstack is in the path, not the domain: `quay.io/jetstack/cert-manager-controller`.

The simplest solution to *this* problem is to publish the images to `quay.io/cert-manager/cert-manager-controller`,
which we now have the ability to do.

Reply:
I agree. The design reads like the number one benefit is to "become vendor neutral", but we have established that Scarf isn't required for that. I suggest that we don't mention this benefit in the design document at all.

The number one benefit that would justify Scarf's adoption, to me, being able for maintainers to know each version's adoption... and also being able to show a daily number of downloads at events like KubeCon.